packages/expect/src/jasmineUtils.ts

```diff
@@ -107,6 +107,8 @@ function eq(
       return a == String(b);
     case '[object Number]':
       return Object.is(Number(a), Number(b));
+    case '[object BigInt]':
+      return Object.is(BigInt(a), BigInt(b));
```

Comment:
Good eyes to follow the pattern for Number, but does it make sense that `a === b` is okay for BigInt?

* `Object.is` for `-0` and `NaN` but `0n === -0n` and `Object.is(0n, -0n)` are both `true`
* `Number(…)` cast in #7948 for number object instance, but `new BigInt(…)` throws `TypeError: BigInt is not a constructor`

Reply:
yeah good call, CTRL-C/V inst always efficient 😉